plugin-api/src/main/java/org/hyperledger/besu/plugin/services/query/IbftQueryService.java

```diff
@@ -19,7 +19,15 @@
 
 import java.util.Collection;
 
-/** Allows for the IBFT 2.0 specific aspects of the block chain to be queried. */
+/**
+ * Allows for the IBFT 2.0 specific aspects of the block chain to be queried.
+ *
+ * <p>* @deprecated This interface has been replaced by {@link *
```

Comment:
This comment doesn't seem quite right. I think what you have about the BftQueryService is what should go here.

Reply:
links to the BFTQueryService now (is that what you had in mind?)